execution.bs

```diff
@@ -5011,6 +5032,17 @@ enum class forward_progress_guarantee {
         };
     </pre>
 
+2. Given a subexpression `sndr`, let `Sndr` be `decltype((sndr))`, let `Env` be
```

Comment:
i've been trying to converge on snake_case for variables/subexpressions, PascalCase for types, and ALLCAPS for macros and pseudo-macros. It would be nice if the library wording as a whole settled on a style guide.